lib/src/blocs/new_post/new_post_event.dart

```diff
@@ -22,24 +24,28 @@ class EditPost extends NewPostEvents {
   final String postText;
   final String postId;
   final List<TopicUI> selectedTopics;
+  final bool isRepost;
 
   EditPost({
     required this.postText,
     this.attachments,
     required this.postId,
     required this.selectedTopics,
+    this.isRepost = false,
   });
 }
 
 class DeletePost extends NewPostEvents {
   final String postId;
   final String reason;
   final int? feedRoomId;
+  final bool isRepost;
```

Comment:
remove this var if not needed

Reply:
we can not remove it from here since we are not getting attachments here